src/main/java/com/vestmark/nexus/plugin/maven/MavenResource.java

```diff
@@ -184,6 +184,11 @@ public Response download(
       response.header(HttpHeaders.CONTENT_DISPOSITION, String.format("attachment;filename=\"%s\"", fileName));
       return response.build();
     }
+    catch (Exception e) {
+      log.debug("Exception: {}", e.toString());
```

Comment:
`log.error("Error: ", e);`
Replace error with your message of choice.  This will ensure we get the stack trace as @k-brooks mentioned.